### PR TITLE
Update Slack invite link

### PR DESCRIPTION
**Pull request by RachL**
#### What? Why?

Updated Slack invite link in README

#### Release notes

Slack invite link is fixed in the README file.

Changelog Category:  Fixed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Supported by the Open Food Foundation and a network of global affiliates, we are
 
 We're part of global movement - get involved!
 
-* Join the conversation [on Slack][slack-invite]. Make sure you introduce yourself in the #general channel
+* Join the conversation [on Slack](https://join.slack.com/t/openfoodnetwork/shared_invite/enQtMzU2Mjk5MDc2MjA5LTM4ZTAzZjIwNzIxMmU5ODFiNWY1MTU2ZWUyNzQwNjdjNTY0N2VhY2UwOGU4ZmVjNzYyZDU2NjY3NzZkZmQwYjk). Make sure you introduce yourself in the #general channel
```
Comment (sigmundpetersen) [inline README.md]: I wonder if you should edit the `slack-invite` reference at the bottom rather than the link directly? 🙂

Comment (sauloperez) [reply README.md]: Right. Check reference-style at https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Comment (RachL) [reply README.md]: Right, thx Sigmund, I completely missed that one! I've added a commit to fix this

**Comment by sauloperez** [comment]
Awesome @RachL !

**Pull request merged**